### PR TITLE
Updating to pypi sphericart release

### PR DESCRIPTION
**Pull request by agoscinski**
@frostedoyster do you have a pure cpu implementation of spherical harmonics using torch tensors? I cannot run this on my laptop because for the sphericart torch version I need CUDA, and the sphericart numpy version will cut off autograd

### CODE REVIEW

**Comment by agoscinski** [comment]
Ah I found something in the sphericart project, seems more like a problem of the installation setup

**Comment by agoscinski** [comment]
@frostedoyster Tried to update sphericart, but the output form has change, so I get an error when running the current version on this branch. Can we solve this tomorrow out?

```bash
  File "/home/alexgo/code/torch_spex/torch_spex/spherical_expansions.py", line 133, in forward
    densities_l.index_add_(dim=0, index=density_indices.to(expanded_vectors_l.device), source=expanded_vectors_l)
RuntimeError: index_add_(): self (Float) and source (Double) must have the same scalar type
Uncaught exception. Entering post mortem debugging
Running 'cont' or 'step' will restart the program
> /home/alexgo/code/torch_spex/torch_spex/spherical_expansions.py(133)forward()
-> densities_l.index_add_(dim=0, index=density_indices.to(expanded_vectors_l.device), source=expanded_vectors_l)
```

**Comment by agoscinski** [comment]
Okay had time to look at the error messages, it was trivial fix, runs now

File: torch_spex/spherical_expansions.py [hunk 1]
```diff
@@ -146,6 +146,7 @@ def forward(self, structures):
                     where_ai = torch.LongTensor(np.where(ai_new_indices == a_i)[0]).to(densities_l.device)
                     densities_ai_l = torch.index_select(densities_l, 0, where_ai)
                     labels.append([a_i, l, 1])
+                    breakpoint()
```
Comment (frostedoyster) [inline torch_spex/spherical_expansions.py:149]:
```suggestion

```

**Review by frostedoyster — COMMENTED** [review]
Thanks! Just a stray breakpoint

File: torch_spex/spherical_expansions.py (hunk 1 above)
Comment (agoscinski) [reply torch_spex/spherical_expansions.py]: thanks forgot this one

**Review by frostedoyster — APPROVED** [review]

**Pull request merged**